README.md

```diff
@@ -1,6 +1,14 @@
 # ci-imgs
 
-This repository contains the Dockerfile for the CI images used by RAPIDS. These images are built from the [rapidsai/mambaforge-cuda](https://github.com/rapidsai/mambaforge-cuda) images.
+This repository has been merged with [cibuildwheel-imgs](https://github.com/rapidsai/cibuildwheel-imgs). This repository used to contain only the dockerfile used for the CI images used by RAPIDS (these images are build from the [rapidsai/mambaforge-cuda](https://github.com/rapidsai/mambaforge-cuda) images).
+
+Added now to this repository are all the images used to build [RAPIDS pip wheel](https://rapids.ai/pip) releases.
```

Comment:
```suggestion
This repository includes the following CI images for `RAPIDS`:
```

The `cibuildwheel-imgs` repository will be archived after this PR is merged.

Therefore, we don't need to link to it since this repository will be the new source of truth.